.github/workflows/tag-and-release.yml

```
@@ -0,0 +1,124 @@
+# This creates release on pushing with tag
+
+name: tag-and-release
+
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+defaults:
+  run:
+    working-directory: cli
+
+jobs:
+  createrelease:
+    name: createrelease
+    runs-on: [ubuntu-latest]
+    outputs:
+      release_url: ${{ steps.create_release.outputs.upload_url }}
+      version: ${{ steps.get_version.outputs.VERSION }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+        shell: bash
+      - name: create release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.get_version.outputs.VERSION  }}
+          release_name: Release ${{ steps.get_version.outputs.VERSION  }}
```

```suggestion
          release_name: ${{ steps.get_version.outputs.VERSION  }}
```

no need to have "Release" in the Release.